ESSArch_Core/ip/tests/test_models.py

```diff
@@ -166,3 +168,117 @@ def test_status_is_100_if_state_is_an_unhandled_type(self):
         self.ip.state = uuid.uuid4
 
         self.assertEqual(self.ip.status(), 100)
+
+
+class InformationPackageStepStateTests(TestCase):
+
+    class MockedTask(object):
```

Comment:
Is this used or can it be removed?

Reply:
Not used, seems I trust PyCharm to much! :D 
Will remove.